src/test/java/math/Matrix4x4Test.java

```diff
@@ -0,0 +1,51 @@
+package math;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class Matrix4x4Test {
+    Matrix4x4 m1 = new Matrix4x4(new float[][]{
+            {1,0,0,1},
+            {2,0,1,0},
+            {3,0,5,0},
+            {0,1,2,2}
+    });
+    Matrix4x4 m2 = new Matrix4x4(new float[][]{
+            {3,1,1,0},
+            {5,0,0,1},
+            {4,0,1,5},
+            {3,1,5,6}
+    });
```

Comment:
Create `@beforeAll` init function and put all initialization there. It's important to make sure that one test does not change input data and affect others.

~~~
@beforeAll
void init() {
  ....
  }
  ~~~